package.json

```diff
@@ -21,8 +21,7 @@
     "build": "yarn run clean && tsc -p ./tsconfig.es2015.json && tsc -p ./tsconfig.cjs.json",
     "watch": "tsc -w -p ./tsconfig.es2015.json",
     "storybook": "start-storybook -p 6006",
-    "storybook:build": "build-storybook",
-    "prepare": "yarn run build"
+    "storybook:build": "build-storybook"
```

Comment:
Please bump the version to "3.9.0-alpha".

Reply:
done